config/starship.toml

```
@@ -0,0 +1,22 @@
+# Starship configuration
+# This is a placeholder configuration file for Starship prompt
+# Add your Starship configurations here
+
+format = """
+$username\
+$hostname\
+$directory\
+$git_branch\
+$git_state\
+$git_status\
+$cmd_duration\
+$line_break\
+$python\
+$character"""
+
+[directory]
+style = "blue"
+
+[character]
+success_symbol = "[➜](bold green)"
+error_symbol = "[➜](bold red)"
```

Change the content to the following:
```suggestion
"$schema" = 'https://starship.rs/config-schema.json'

add_newline = true

[character]
success_symbol = "[➜](bold green)"
error_symbol = "[➜](bold red)"

[package]
disabled = true
```